.takt/facets/instructions/validate-impl.md

```
@@ -0,0 +1,23 @@
+実装が要件・設計・タスクに合致しているかを検証せよ。
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)